docs/extend/README.md

```diff
@@ -1,6 +1,11 @@
 # Extend
 
+Matestack provides you with a lot of functionality out of the box. But at certain points, you probably need to go your own individual way. Thankfully, the `matestack-ui-core` gem can be easily extended!
+
 ## Custom Components
 
-1. Create custom static Components (coming soon)
-2. Create custom dynamic Components (coming soon)
+See below for a list of guides on how to create your own project-specific custom components:
+
+1. [Custom static components](/docs/extend/custom_static_components.md)
+2. [Custom dynamic components](/docs/extend/custom_dynamic_components.md)
+3. [Custom actionview components](/docs/extend/custom_actionview_components.md)
```

Comment:
I don't like to see the actionview component on the same hierarchy level as the static and dynamic components. the actionview component is just a variant of either the static or dynamic component. I would go for an extra section in this overview, describing the role of actionview components and than link to the documentation OR add a section to both the docs of static and dynamic components

Reply:
I don't want the users to think, that there are THREE component concepts in matestack. There are only TWO important ones to understand ;)

Reply:
True, I've changed them as sub-items (static/dynamic) and added a link to the component concept explanation